compressor/conf.py

```diff
@@ -70,6 +70,12 @@ class CompressorConf(AppConf):
     OFFLINE_MANIFEST = 'manifest.json'
     # The Context to be used when TemplateFilter is used
     TEMPLATE_FILTER_CONTEXT = {}
+    # URL-less mode: you do offline compression & decompression regardless of settings.STATIC_URL
+    # and settings.COMPRESS_URL so your assets become re-locatable for any URLs.
+    # Affects manifest file contents and manifest key determination, but not the assets contents.
+    OFFLINE_URLLESS = False
```

Comment:
IMO, the placeholder is a technical detail of `STATIC_URL`-independent mode implementation :) 

Nobody should care how is it implemented: using placeholders or by generating manifest in absolutely different way.